src/modules/common/components/Page.js

```diff
@@ -54,13 +57,16 @@ class Page extends React.Component<PropType> {
   }
 
   render () {
+    const {title, children, theme, content, files} = this.props
     return (
       <>
-        <WebContainer theme={this.props.theme}>
+        <Caption title={title} />
+        {children}
```

Comment:
Children can have arbitrary heights. So this will not work with together with the webview.  Maybe a refactor to `ContentView` which renders the webview and a component for each different usage  of `ContentView`  would make sense.

So you would have a EventPage and CategoriesPage which both render a `ContentView` ContentView would tage a required height.

Reply:
How exactly does the webview render? I don't quite understand why it shouldn't just move down automatically, I mean everything is in a ScrollView and has a 'column' flex direction.  
What do you mean by passing a height to the content view? How would we know how much space a specific component needs on each device?

Reply:
Each component next to a WebView needs to have a static height. The WebView brings its own scrolling behaviour. Mixing it with a ScrollView does not work afaik.

Reply:
The height of the device specific statusbar can be calculated.

Reply:
I think only this needs to changed here. Else this is ready for merge.

You can also create a separate task to figure out a way to handle the webview.

Reply:
I'll have a look at it in NATIVE-32.